tests/test_translator.py

```diff
@@ -277,6 +281,232 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        # Arguments
+        args = argparse.Namespace(
```

Comment:
I would completely agree with you on that point. but still the tests shouldn't depend on them, and my aim was to just lay down some cases. Putting some utility functions like ``namespace_loader`` should make these tests independent of the way anything's written. Moreover, as far as your reason is concerned to make it go through the parser. the ``namespace_loader`` can be written in such a way that it goes through the parser as well. Modularizing the things will definitely help.